spec/cucumber/core/test/result_spec.rb

```diff
@@ -27,7 +32,11 @@ module Cucumber::Core::Test
         expect( result.duration ).to eq duration
       end
 
-      it "requires the constructor argument" do
+      it "has embeddings" do
+        expect( result.embeddings ).to eq embeddings
+      end
+
+      it "requires the first constructor argument" do
```

Comment:
should there be a sister test to this now that says embeddings are blank by default? If you're trying to keep it symmetrical.

EDIT: Same for the test below that requires 2 args

Reply:
I'm actually tempted to remove that test as it basically tests a setter+getter.
And I've not been able to find a way to keep things DRY and symmetrical without being overkill.

Reply:
Re-review. Feedback needed here. What are your thoughts / plans?

Reply:
I've found how to DRY those tests using shared_examples.